tests/integration_test/services_test.py

```
@@ -58,6 +66,14 @@
 from kairon.shared.cognition.data_objects import CognitionSchema, CognitionData, CollectionData
 from kairon.shared.constants import EventClass, ChannelTypes, KaironSystemSlots
 from kairon.shared.data.audit.data_objects import AuditLogData
+import pytest
```

Ruff (0.12.2)

Redefinition of unused `pytest` from line 17

Remove definition: `pytest`

(F811)